cranelift/filetests/filetests/isa/x64/cmp-mem-bug.clif

```diff
@@ -60,10 +60,10 @@ block0(v0: f64, v1: i64):
 ;   setz    %al
 ;   andl    %edi, %eax, %edi
 ;   movzbq  %dil, %rax
-;   ucomisd %xmm0, %xmm9
+;   ucomisd %xmm9, %xmm0
 ;   movdqa  %xmm0, %xmm2
-;   mov z, sd; j%xmm2 $next; mov%xmm0 %xmm0, %xmm0; $next: 
 ;   mov np, sd; j%xmm2 $next; mov%xmm0 %xmm0, %xmm0; $next: 
+;   mov z, sd; j%xmm2 $next; mov%xmm0 %xmm0, %xmm0; $next: 
```

Comment:
Pre-existing, but I think something is out-of-order in the pretty-printing here -- would you mind fixing this while we're here?